messaging/kafka-avro-reactive-messaging/src/test/java/io/quarkus/ts/openshift/messaging/kafka/resources/containers/SchemaRegistryContainer.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.ts.openshift.messaging.kafka.resources.containers;
```

Comment:
I think that "resources.containers" is too much, with just "containers" is clear enough.